app/forms/recaptcha_form.rb

```diff
@@ -107,6 +109,10 @@ def is_result_error?(error_code)
     RESULT_ERRORS.include?(error_code)
   end
 
+  def result_reason_exempt(result)
```

Comment:
since this returns a boolean, seems like a good candidate for a `?`
```suggestion
  def result_reason_exempt?(result)
```